locales/en/plugin__kubevirt-plugin.json

```diff
@@ -876,6 +876,7 @@
   "Terms": "Terms",
   "The changes you have made require this VirtualMachine to be restarted.": "The changes you have made require this VirtualMachine to be restarted.",
   "The channel to track and receive the updates from.": "The channel to track and receive the updates from.",
+  "The DataImportCron polls disk images and imports the images as persistent volume claims. You can configure the image source and other settings on the DataImportCron details page.": "The DataImportCron polls disk images and imports the images as persistent volume claims. You can configure the image source and other settings on the DataImportCron details page.",
   "The Descheduler can be used to evict a running VirtualMachine so that the VirtualMachine can be rescheduled onto a more suitable Node via a live migration.": "The Descheduler can be used to evict a running VirtualMachine so that the VirtualMachine can be rescheduled onto a more suitable Node via a live migration.",
```

Comment:
I'm not sure what "can be used" means in this context. Are you implying that the Descheduler has other uses? 

Suggestion: "The Descheduler evicts a running VirtualMachine so that the VirtualMachine can be rescheduled on a more suitable Node by using a live migration."

I know that "by using" is more verbose than "via" but our doc guidelines do not allow Latin words. Guidelines also say that "by using" is better than "using" because it creates fewer problems for localization.

Reply:
I will try to review the other lines when I have time.